### PR TITLE
refactor(banner): sync storybook and Volto component

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Comment by nileshgulia1** [comment]
@eea-jenkins build storybook

**Comment by nileshgulia1** [comment]
@eea-jenkins build storybook

**Comment by nileshgulia1** [comment]
@eea-jenkins build storybook

**Comment by nileshgulia1** [comment]
@eea-jenkins build docusaurus

**Comment by nileshgulia1** [comment]
@eea-jenkins build storybook

**Comment by nileshgulia1** [comment]
@eea-jenkins build storybook

**Comment by nileshgulia1** [comment]
@eea-jenkins build all

File: src/ui/Banner/Banner.jsx [hunk 1]
```diff
@@ -1,22 +1,68 @@
...
+    shareLink: (url) =>
+      `https://www.linkedin.com/sharing/share-offsite/?url=${url}`,
+  },
+  reddit: {
+    shareLink: (url, title) => `https://reddit.com/submit?url=${url}`,
+  },
+};
+
+export const getImageSource = (image) => {
+  if (image?.download) return image.download;
```
Comment (ichim-david) [inline src/ui/Banner/Banner.jsx:29]: @nileshgulia1 with this pull request the homepage gif is no longer rendering.
This is what the URL is rendered with this branch which is incorrect
background-image:url(https://demo-www.eea.europa.eu/en/jumping-bug.gif)
This is what is currently which works
background-image: url("https://demo-www.eea.europa.eu/en/jumping-bug.gif/@@images/image");

**Review by ichim-david — CHANGES_REQUESTED** [review]

File: src/ui/Banner/Banner.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/ui/Banner/Banner.jsx:29]: @ichim-david You need to test this PR with https://github.com/eea/volto-eea-website-theme/pull/76. Sorry for not mentioning beforehand.

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Pull request merged**